docusaurus.config.js

```diff
@@ -214,7 +214,7 @@ const config = {
       },
       algolia: {
         appId: "ZXLX1PN8SG",
-        apiKey: "de4ef56c89d1bc997b27afd35c466672",
+        apiKey: "abad5ee4b6f688bf466290763737a957",
```

Comment:
BTW normally its not safe to keep API keys here as its a public repo & anyone can access or misuse it. For now, you're only updating it, so it makes sense to keep up the flow. But to be more secure, I think we should keep such keys in Github Secrets.

Reply:
Yes, I will dive into this a bit more.
But in Algolia, it's stated that this API key is the "search-only" key, which is safe to be used in FE.